wings/market/market_base.pyx

```diff
@@ -37,6 +37,10 @@ cdef class MarketBase(TimeIterator):
         for event_tag in self.MARKET_EVENTS:
             self.c_add_listener(event_tag.value, self.event_reporter)
 
+    @property
+    def market_name(self) -> str:
```

Comment:
Yvonne's performance MVP patch has a similar proposal as well. You may want to merge with hers on this. I like the shorter `names` property better.

Reply:
Okay I will change to "name".